### ISSUE TITLE
Debug logging not displaying the prebid version since 2.0

### ISSUE THREAD

**Issue by andyblackwell** [prebid/Prebid.js#3997]
## Type of issue
Bug

## Description
After version 2.0 the prebid version is not being logged to the console

## Steps to reproduce
Import the project as suggested in the example available in http://prebid.org/dev-docs/examples/basic-example.html
and notice that e.g. "INFO: Prebid.js v2.18.0 loaded" is not showing.

## Other information
dupe of closed bug #3700 
This issue was fixed in #3714 but then the fix was reverted in #3736 

my opinion: setConfig() calls should not wait until pbjs.processQueue() is called

**Comment by snapwich** [comment]
Did you reference the correct pull-request?  I don't see the revert in 3776.

**Comment by jsnellbaker** [comment]
This was the revert PR https://github.com/prebid/Prebid.js/pull/3736 @snapwich

**Comment by andyblackwell** [comment]
whoops, thanks! updated in description

**Comment by snapwich** [comment]
why was it reverted @jsnellbaker ? there's not much of a description in the pull-request.

**Comment by jsnellbaker** [comment]
I think there were build issues at the time (I think they were in circleci) that resulted from this line of code:
https://github.com/prebid/Prebid.js/pull/3714/files#diff-9d9a6cd82f41984872a66a3ab0d440c4R16

I was trying to recreate/retest the changes yesterday, but I had to change gears after something came up.  I can take another look if you want.

**Comment by snapwich** [comment]
yeah that'd be great.  I don't see why that change should cause any issues and I don't think it would have merged if it was causing CI issues.  but if it is causing issues, it'd be nice to know what they are so I can fix them.

**Comment by jsnellbaker** [comment]
@snapwich I setup the changes again in this branch `debug_updates`.  The build compiles fine and works successfully on test pages.  The circleci tests also pass fine (which I saw after creating the above PR).

However when digging more into my past messages, I found the cause for the revert.  When running `gulp test-coverage` with these changes, it causes a number of exception errors to be thrown.  

Below is a copy of one of the messages:
```
SourceMap position not found for trace: {
  "message": "Uncaught TypeError: Cannot read property 'f' of undefined\nat http://localhost:9876/base/test/test_index.js?d15ff8c5eaddb452b39bc74c8b979a6a8bd1bafd:184:189100\n\nTypeError: Cannot read property 'f' of undefined\n    at Object.getParameterByName (http://localhost:9876/base/test/test_index.js?d15ff8c5eaddb452b39bc74c8b979a6a8bd1bafd:184:189100)\n    at Object.<anonymous> (http://localhost:9876/base/test/test_index.js?d15ff8c5eaddb452b39bc74c8b979a6a8bd1bafd:247:41716)\n    at __webpack_require__ (http://localhost:9876/base/test/test_index.js?d15ff8c5eaddb452b39bc74c8b979a6a8bd1bafd:20:30)\n    at Object.<anonymous> (http://localhost:9876/base/test/test_index.js?d15ff8c5eaddb452b39bc74c8b979a6a8bd1bafd:176:66)\n    at __webpack_require__ (http://localhost:9876/base/test/test_index.js?d15ff8c5eaddb452b39bc74c8b979a6a8bd1bafd:20:30)\n    at Object.<anonymous> (http://localhost:9876/base/test/test_index.js?d15ff8c5eaddb452b39bc74c8b979a6a8bd1bafd:2790:65)\n    at __webpack_require__ (http://localhost:9876/base/test/test_index.js?d15ff8c5eaddb452b39bc74c8b979a6a8bd1bafd:20:30)\n    at Object.<anonymous> (http://localhost:9876/base/test/test_index.js?d15ff8c5eaddb452b39bc74c8b979a6a8bd1bafd:5392:71)\n    at __webpack_require__ (http://localhost:9876/base/test/test_index.js?d15ff8c5eaddb452b39bc74c8b979a6a8bd1bafd:20:30)\n    at Object.<anonymous> (http://localhost:9876/base/test/test_index.js?d15ff8c5eaddb452b39bc74c8b979a6a8bd1bafd:19643:1)",
```

Would you be able to help take a look?

**Comment by snapwich** [comment]
I'll dig in.  We're definitely experiencing the same thing as this: https://github.com/webpack-contrib/istanbul-instrumenter-loader/issues/83

Although that has no fix.  But at least we know what tool the issue is coming from.

**Comment by snapwich** [comment]
should be fixed by #4007 , let us know if that is not the case.
